plugins/codescene/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint')],
+};
```

Comment:
nowadays, the eslint files all [look like this](https://github.com/backstage/backstage/blob/f28947a4f8dded92292f277aae32749b0af7fe80/packages/catalog-model/.eslintrc.js) - this automatically lints the correct way depending on what the package role is.